src/core/evaluator.js

```diff
@@ -3612,7 +3612,16 @@ class TranslatedFont {
             resources: fontResources,
             operatorList,
           })
-          .then(function () {
+          .then(() => {
+            // According to the PDF specification, section "9.6.5 Type 3 Fonts"
+            // and "Table 113":
+            //  "A glyph description that begins with the d1 operator should
+            //   not execute any operators that set the colour (or other
+            //   colour-related parameters) in the graphics state;
+            //   any use of such operators shall be ignored."
+            if (operatorList.fnArray[0] === OPS.setCharWidthAndBounds) {
+              this._removeType3ColorOperators(operatorList);
```

Comment:
The spec also mentions we should ignore some `gs` entries. It looks like the only one we support is `TR`, but it shouldn't be too hard to add an option to the evaluator to skip that or remove it here.

Reply:
> The spec also mentions we should ignore some `gs` entries.

The reason that I didn't add this originally was because I had no test-case where it mattered, and it seemed easier to just ignore it.

> It looks like the only one we support is `TR`, but it shouldn't be too hard to add an option to the evaluator to skip that or remove it here.

The updated patch ignores all listed types, not only `TR`, to prevent a future bug if/when we support more of these.

Also, given that this only applies to Type3 glyphs I didn't want add special-casing to the general GState handling, which is why I opted to simply remove them after the fact here.